components/eamxx/scripts/test-all-scream

```diff
@@ -133,9 +147,6 @@ OR
                         help="Set the testing level. Defaults to medium unless the test is cov or mem_check"
                         "(short is default in those cases).")
 
-    parser.add_argument("--force-baseline-regen", action="store_true",
```

Comment:
Does this mean that if the user passes `-g`, we update the baselines regardless of how the current SHA compares to the baselines' SHA?

Reply:
Yes.